api/validation/swagger_api_validation.yml

```diff
@@ -107,12 +107,13 @@ components:
         operator:
           type: string
         datetime:
-          type: integer
-          format: int64
+          type: string
+          format: date-time
         status:
           type: string
         proofData:
           type: string
+          format: byte
```

Comment:
byte? Type is string... Did you intend to write something like base64 or something like that?

Reply:
https://swagger.io/docs/specification/data-models/data-types/  (check String formats)
A base64 string is a string with format byte